src/luaot.c

```diff
@@ -807,3 +856,12 @@ void print_source_code()
 
     fclose(infile);
 }
+
+static
+void print_source_code(lua_State *L)
+{
+    if (precompile)
+        print_source_code_bytecode(L);
+    else
+        print_source_code_source(L);
```

Comment:
Please use braces around the if/else bodies.

Reply:
will do